library/core/src/slice/index.rs

```diff
@@ -13,7 +13,7 @@ where
 {
     type Output = I::Output;
 
-    #[inline]
+    #[inline(always)]
```

Comment:
I needed to apply this to fix the test added in https://github.com/rust-lang/rust/pull/119878, I don't know if this deserves a comment? And if it does, I'm not sure if I should cite the test specifically; that seems rather fragile if the test is renamed.

Reply:
That PR did have the always too so it must have been removed in the meantime.

Reply:
That PR added an always to `Result::unwrap`, this one is in the `Index` impl for slices. The assertion in this PR blows up the size of the MIR for slice indexing, so I think LLVM needs the `always` boost. But this function is also a trivial wrapper, so it's reasonably defensible in any case.